### DISCUSSION TITLE
How to add plugins in HTML (not node).

### DISCUSSION THREAD

**Discussion by Inotart** [sourcelair/xterm.js#4449] [Q&A] [answered]
I am a child who does not use node, I directly load xterm .js this library according to the official default demonstration. And part of the code that calls xterm is written in HTML. Ask for advice
![image](https://user-images.githubusercontent.com/101685021/227789898-d08152b2-6ec6-44a4-99eb-009e60762cd7.png)

**Comment by Tyriar** [comment, marked as answer]
I believe the addon gets added on to `window`, something like:

```js
var fitAddon = newwindow.FitAddon.FitAddon()
```

**Reply by Inotart** [reply]
Thanks, it's normal now

**Comment by Inotart** [comment]
你好,我目前不在，如果你只是看到论坛邮件就不用回复了
